recipes/source_location/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+import os
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class SourceLocationConan(ConanFile):
+    name = "source_location"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Rechip/source_location"
+    description = "source_location header for some older compilers"
+    topics = ("cpp", "source_location", "header-only")
+    settings = ["compiler"]
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  strip_root=True, destination=self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses",
+                  src=self._source_subfolder)
+        self.copy(pattern="*", dst="include",
+                  src=os.path.join(self._source_subfolder, "include"))
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "16.6",
+            "gcc": "7.1",
+            "clang": "9",
+            "apple-clang": "12",
+        }
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
```

Comment:
```suggestion
        if self.settings.compiler.get_safe("cppstd"):
```
Please get cppstd property safely.